coinlib_flutter/README.md

```diff
@@ -26,4 +26,4 @@ An example app is provided in `example/` that demonstrates use of the loader
 widget. Beyond this, the [coinlib](https://pub.dev/packages/coinlib) library
 documentation can be followed.
 
-Android, iOS, Linux, macOS and Web are supported.
+Android, iOS, Linux, macOS, web, and Windows are supported.  If you are using the library for web, Android, or iOS, the library is ready to use.  For desktop platforms, see [coinlib's documentation](https://pub.dev/packages/coinlib) for instructions on building the native library.
```

Comment:
"For desktop platforms" ought to be "For Windows". Perhaps here direct instruction on building the library can be included. I guess `dart run coinlib:build_windows` (or wsl) should be run and then the flutter CMake build will pick up the library from the local build folder?

Reply:
Resolved by 8d350c26ae91a2a355e5d1e7e2e189582428c273

Is 2.0 automatically built for linux et al?  We're using 1.0 right now and I thought we had to construct coinlib for the other desktop platforms.